programming basics/Python Built-ins.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>I'm guessing below was not meant to be there, since this comes below in the looping section.</p><pre class="ql-syntax" spellcheck="false"># print out the even numbers from 1-25
# if the number is not even, do nothing
conditional_statement = True
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>